vws.gemspec

```diff
@@ -20,13 +20,13 @@ Gem::Specification.new do |spec|
 
 
   #development depedencies
-  spec.add_development_dependency "bundler", "~> 1.9.1"
+  spec.add_development_dependency "bundler"
   spec.add_development_dependency "rake", "~> 10.4.2"
   spec.add_development_dependency "rspec","~> 3.2.0"
   #spec.add_development_dependency "json","~> 1.8"
 
   #runtime dependencies
-  spec.add_runtime_dependency "rest-client", "~> 1.8"
+  spec.add_runtime_dependency "rest-client", "~> 2.0.2"
```

Comment:
I was running into mime-types compatible issues because rest-client version was too old.

> Bundler could not find compatible versions for gem "mime-types":
> In snapshot (Gemfile.lock):
> mime-types (= 3.2.2)
> 
> In Gemfile:
> carrierwave-google-storage was resolved to 0.8.0, which depends on
> carrierwave (~> 1.3.1) was resolved to 1.3.1, which depends on
> mime-types (>= 1.16)
> 
> ```
> file_validators was resolved to 2.3.0, which depends on
>   mime-types (>= 1.0)
> 
> carrierwave-google-storage was resolved to 0.8.0, which depends on
>   google-cloud-storage (~> 1.17.0) was resolved to 1.17.0, which depends on
>     google-api-client (~> 0.26) was resolved to 0.28.4, which depends on
>       mime-types (~> 3.0)
> 
> vws was resolved to 1.2.5, which depends on
>   rest-client (~> 1.8) was resolved to 1.8.0, which depends on
>     mime-types (< 3.0, >= 1.16)
> ```
> Running `bundle update` will rebuild your snapshot from scratch, using only
> the gems in your Gemfile, which may resolve the conflict.